vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ContentBlockManager.java

```diff
@@ -201,6 +223,49 @@ public void deleteTextBlockById(String blockId, String slideId) throws BlockDoes
         }
 
     }
+    
+    @Override
+    public ISpace getCurrentSpaceForSpaceBlock(String blockId, String slideId) throws BlockDoesNotExistException {
```

Comment:
just noticing this now, why is this a method? Why not just get the spaceblock in the controller and then call `getSpace`? That's what it's therefore after all.